_tuning-your-cluster/replication-plugin/getting-started.md

```diff
@@ -263,7 +263,7 @@ curl -XGET -k -u 'admin:admin' 'https://localhost:9200/_plugins/_replication/fol
 When you're done making changes, resume replication:
 
 ```bash
-curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:admin' 'https://localhost:9200/_plugins/_replication/follower-01/_resume?pretty' -d '{}'
+curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:< Admin password >' 'https://localhost:9200/_plugins/_replication/follower-01/_resume?pretty' -d '{}'
```

Comment:
```suggestion
curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:<custom-admin-password>' 'https://localhost:9200/_plugins/_replication/follower-01/_resume?pretty' -d '{}'
```

Reply:
Applied suggestion in: [cdeb90f](https://github.com/opensearch-project/documentation-website/pull/5887/commits/cdeb90ffae8be7a3ae2f9de267abc08879dd44cd)